package.json

```diff
@@ -114,9 +113,8 @@
   },
   "peerDependencies": {
     "moment": "^2.18.1",
-    "react": ">=0.14",
-    "react-dom": ">=0.14",
-    "react-addons-shallow-compare": ">=0.14"
+    "react": ">=15.3.0",
+    "react-dom": ">=15.3.0"
```

Comment:
I'm not personally comfortable with dropping react 14 support yet (that would be a semver major).

 Unless we want to implement a detection mechanism, such that we only inherit from PureComponent in 15.3+, I don't think this is a change we should make.